File: client_seller_web/src/main/webapp/js/service/hallService.js
//服务层
app.service('hallService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../hall/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../hall/findPage.do?pageNum='+page+'&pageSize='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../hall/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../hall/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../hall/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../hall/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../hall/search.do?pageNum='+page+"&pageSize="+rows, searchEntity);
	}    	
});

File: client_manage_interface/src/main/java/cn/edu/aynu/rjxy/management/service/HallService.java
package cn.edu.aynu.rjxy.management.service;


import cn.edu.aynu.rjxy.pojo.TbHall;
import com.github.pagehelper.PageInfo;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface HallService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbHall> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageInfo findPage(int pageNum, int pageSize);


	/**
	 * 增加
	*/
	public void add(TbHall hall);


	/**
	 * 修改
	 */
	public void update(TbHall hall);


	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbHall findOne(Long hallId);


	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] hallIds);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageInfo findPage(TbHall hall, int pageNum, int pageSize);
	
}
